lib/rest/routes/credentials.js

```diff
@@ -1,8 +1,6 @@
-
 const express = require('express');
 const credentialSrv = require('../../services').credential;
-const usersSrv = require('../../services').user;
-const appsSrv = require('../../services').application;
+const findConsumer = require('../utils/findConsumer');
```

Comment:
why do you need some util if you can put the logic directly into service ?

Reply:
This should now be fixed in 96783ca

Reply:
Uh actually no. Thing is `findConsumer` is to find an `User` or an `Application`.

In which service should I put this? We do not have a Consumer service

Reply:
id or username resolution -> user service 
id or app name -> app service 

user or app? 
isn't it the only place find consumer is needed ? if so I would keep it there

Reply:
Yes, it' the only place. Let's keep it there for now then.